client/components/domains/contact-details-form-fields/test/index.js

```diff
@@ -216,9 +245,11 @@ describe( 'ContactDetailsFormFields', () => {
 				userCountryCode: 'NZ',
 			};
 
-			const wrapper = shallow( <ContactDetailsFormFields { ...newProps } /> );
+			render( <ContactDetailsFormFields { ...newProps } /> );
 
-			expect( wrapper.find( 'FormPhoneMediaInput' ).props().countryCode ).toEqual( 'JP' );
+			expect( FormPhoneMediaInput.mock.calls[ 0 ][ 0 ] ).toEqual(
```

Comment:
Could we use `toHaveBeenCalledWith()` instead? Or for better specificity, `toHaveBeenNthCalledWith()`

Same question for other instances where we directly access `.mock.calls`

Reply:
`.toHaveBeenCalledWith` doesn't seem to work with `expect.objectContaining` in that case. I'm not exactly sure why but using `mock.calls` directly fixed that.

Reply:
How come? What's the difference with, say, https://github.com/Automattic/wp-calypso/blob/ba53a75e685eef77a424bebc0bdccea9e9b55c7f/client/components/forms/form-phone-input/test/index.jsx#L65?

Reply:
Seems like those assertions need a second argument when called with a React component. Fixed it in https://github.com/Automattic/wp-calypso/pull/65084/commits/03e569352ec2416edc88eca22cde235ce259f923.